Update Authentication to include incorrect email address error

ruby0606 opened the pull request:
When a user tries to login with an email address with spaces, the error message must display easy to read message:  "The email address may not contain spaces"

ruby0606 committed:
Update Authentication to include incorrect email address error

When a user tries to login with an email address with spaces, the error message must display easy to read message:  "The email address may not contain spaces"

billev reviewed (changes requested):
They've already signed up with their email address, and that email address couldn't have contained spaces. So it's not that their email has a space, but that they didn't enter their email address.

What if we say "Please enter a valid email address" or something along those lines, and set the focus back to the email field?

Set the focus with:
                    $('#username', $dialog).focus();

ruby0606 committed:
Update Authentication file to include invalid email address error on login

When the username contains a space, the error message displayed is not clear to the user. Message is updated to "Invalid email address"
Focus is set on username.

ruby0606 commented:
I changed the text to "Invalid email address" to make the message more clear and set focus on username.
